### ISSUE TITLE
Does this work for WebOS 3.0?

### ISSUE THREAD

**Issue by JerrBear7** [PatrickRorth/Twitch.TV#19]
I want to know before I plug the USB in my TV.

**Comment by Deph0** [comment]
Yes, I've used it on my WebOs 3.0 tv, however you will have to get the WebOSIDE and developer mode app on the tv + make a account on webostv.developer.lge.com to run it, the USB port thing has been closed (afaik, i bought the tv newyears)

**Comment by petroczkii** [comment]
Hey! I have LG B6 with WebOS 3.0. I unziped it to the root of the usb flash drive. I can see the twitch icon at smart menu but nothing happens when i click on it. Any idea?

**Comment by adler-j** [comment]
Any chance for a deeper guide on how to do that @Deph0? It seems the LG webpages for developers are bugged atm (dead links everywhere) so finding out how to do on your own seems hard.

**Comment by petroczkii** [comment]
@adler-j after so many attempts i gave it up and used another app. the Plex has a twitch plugin. try that, it works for me.

**Comment by Deph0** [comment]
@adler-j 
you download the SDK from webostv.developer.lge.com/sdk/

search 'developer' on your tv, download it and enter your account details- use the eclipse IDE to connect to the tv from the bottom right ("target configuration" tab)
